purchase_manual_service_quantity/models/purchase_order_line.py

```diff
@@ -2,8 +2,7 @@
 # Copyright 2016-2017 ACSONE SA/NV
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import api, fields, models, _
-from odoo.exceptions import UserError
```

Comment:
maybe we don't want to remove this import but to raise an exception in _invers_qty_received... To be checked